client/src/components/Poster.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import Poster from './Poster'
+
+export default {
+  title: 'Poster',
+  component: Poster,
+}
+
+const Template = args => <Poster {...args} />
+
+export const Default = Template.bind({})
+Default.args = {}
```

Comment:
I think I'd add the props (path, alt) here as well so you can directly see in storybook that this component gets these props.